src/main/java/com/commercetools/project/sync/state/StateSyncer.java

```diff
@@ -54,7 +75,7 @@ public static StateSyncer of(
   @Override
   protected CompletionStage<List<StateDraft>> transform(@Nonnull List<State> states) {
     return CompletableFuture.completedFuture(
-        StateTransitionReferenceReplacementUtils.replaceStateTransitionIdsWithKeys(states));
+        StateReferenceResolutionUtils.mapToStateDrafts(states));
   }
 
   @Nonnull
```

Comment:
for getQuery, I added a utility into StateReferenceResolutionUtils.

Reply:
https://github.com/commercetools/commercetools-project-sync/pull/158/commits/975155d3130eb8a4cf7a9eace76468c2428ec5e8